docs/CLI.md

```diff
@@ -55,13 +54,15 @@ Command that makes a request to `GET <url>/meta` to fetch the Tool definition an
 
 This returns an OwnerCap object ID that can be used to manage the Tool.
 
-> This command requires that a wallet is connected to the CLI.
+{% hint style="info" %}
+This command requires that a wallet is connected to the CLI..
+{% endhint %}
 
 ---
 
 **`nexus tool register --on-chain <ident>`**
 
-[Issue #96](https://github.com/Talus-Network/nexus-next/issues/96)
+🚧 This is TBD... 🚧
```

Comment:
Same here

Reply:
rephrased in [4f6b9a0](https://github.com/Talus-Network/nexus-sdk/pull/136/commits/4f6b9a079f06a2740aeaaa489dbdd22346192cc3)